frontend/src/App.test.js

```diff
@@ -1,8 +1,11 @@
 import { render, screen } from "@testing-library/react";
 import App from "./App";
 
+
+
 test("renders learn react link", () => {
   render(<App />);
+  
   const title = screen.getByText("Smart Garden Frontend");
   expect(title).toBeInTheDocument();
```

Comment:
Comment this out to pass build tests. Could technically move these over to a Navbar.test.js file